package.json

```diff
@@ -26,6 +26,7 @@
     "ember-cli-app-version": "^1.0.0",
     "ember-cli-dependency-checker": "^1.3.0",
     "ember-cli-eslint": "^3.0.0",
+    "ember-cli-fastboot": "^1.0.2",
```

Comment:
Definitely makes for easier local testing.  Not sure if there are any ramifications to the ember-cli-addon-tests?